packages/TagLabel/TagLabel.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <span class="tag" :class="type">
+    <slot />
+  </span>
+</template>
+
+<script>
+export default {
+  name: "TagLabel",
+  props: {
+    type: { type: String }
+  }
+};
+</script>
+
+<style lang="scss" scoped>
+@import "@mesg-components/theme/_variables";
+
+.tag {
+  display: inline-block;
+  font-size: 12px;
+  line-height: 12px;
+  border-radius: 3px;
+  align-items: center;
+  padding: 6px calc(#{$margin} / 2);
+  color: #0366d6;
+  background-color: rgba(#0366d6, 0.1);
+}
+
+i {
+  font-size: 12px;
+  margin-right: calc(#{$margin} / 4);
+}
```

Comment:
This shouldn't exists. There are no `i` in the component so that's not the right place to put it.